lib/whitelist.js

```diff
@@ -5,7 +5,7 @@ module.exports = settings => (req, res, next) => {
     return matched || d.toLowerCase() === domain.toLowerCase();
   }, false);
 
-  if (isAllowed || !settings.whitelist.length) {
+  if ((isAllowed || !settings.whitelist.length) && !settings.ignore.includes(req.body.to)) {
```

Comment:
I realise this was already there, and it might be just me, but `!settings.whitelist.length` took a second or two to understand - how about
```suggestion
  if ((isAllowed || settings.whitelist.length === 0) && !settings.ignore.includes(req.body.to)) {
```